app/src/main/java/com/duckduckgo/app/browser/DownloadConfirmationFragment.kt

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2019 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.browser
+
+import android.annotation.SuppressLint
+import android.app.Dialog
+import android.content.Intent
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.Toast
+import androidx.core.content.FileProvider.getUriForFile
+import com.duckduckgo.app.browser.downloader.NetworkFileDownloadManager.DownloadFileData
+import com.duckduckgo.app.browser.downloader.NetworkFileDownloadManager.UserDownloadAction
+import com.duckduckgo.app.global.view.gone
+import com.duckduckgo.app.global.view.leftDrawable
+import com.duckduckgo.app.global.view.show
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import kotlinx.android.synthetic.main.download_confirmation.view.*
+import timber.log.Timber
+
+class DownloadConfirmationFragment(
+    private val downloadFileData: DownloadFileData,
+    private val userDownloadAction: UserDownloadAction
+) : BottomSheetDialogFragment() {
+
+    @SuppressLint("RestrictedApi")
+    override fun setupDialog(dialog: Dialog, style: Int) {
```

Comment:
Delete this protected method, use unrestricted onCreateView to setup your layout, something like:
```
override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
      val view = inflater.inflate(R.layout.download_confirmation, container, false)
      setupViews(view)
      return view
}
```